source/includes/_current_enable_status.md

```diff
@@ -0,0 +1,84 @@
+## `/v1/current_enable_status`
+
+> A GET request to this endpoint will return the enable status of all units so that the ERP can sync the unit status
+
+```python
+import requests
+
+url = "https://smartapi.bboxx.co.uk/v1/current_enable_status?page=5&results_per_page=3"
+
+payload={}
+headers = {
+  'Authorization': 'Token token='+A_VALID_TOKEN
+}
```

Comment:
Use same header as in other files, which is this,
`headers = {'Content-Type': 'application/json', 'Authorization': 'Token token=A_VALID_TOKEN'}`